### ISSUE TITLE
In SDA, query patterns should support empty SK queries

### ISSUE THREAD

**Issue by danbdo** [Dynobase/dynobase#163]
If I don't enter (part of) a SK and only provide a PK in the filter pattern the query doesn't work. I get an empty grid. So far, I have to fake that query by setting the SK pattern to "begins with" and leave that empty

**Comment by RafalWilinski** [comment]
Thanks for bug report! To clarify, this relates to single-table designer?

**Comment by danbdo** [comment]
Yeah, no clue what went to my mind when I wrote SDA. Meant to write STD to refer to single-table-designer
